_data/fr.yml

```
@@ -301,16 +301,13 @@ stats:
   hNetworkStats: Stats du réseau
   pSeeMarkets: "<p>Voir <a href='https://bisq.network/markets'>la page des marchés</a> pour le volume par devise.</p>"
   pBSQDistribution: La distribution des BSQ sur le Mainnet est prévue pour avril 2019.
-  pTotalTrades: >
-    <p class='stat-label'>Total des transactions</p> <p class='stat-figure'>32,318</p> <p class='stat-note'>Pour Juin 2019</p>
-  pDailyAverage: >
-    <p class='stat-label'>Moyenne journalière des transactions</p> <p class='stat-figure'>94</p> <p class='stat-note'>En Juin 2019</p>
-  pSoftwareDownloads: >
-    <p class='stat-label'>Téléchargements du logiciel</p> <p class='stat-figure'>13,723</p> <p class='stat-note'>For v1.1.2 (approximation)</p>
-  pGithubContributors: >
-    <p class='stat-label'>Contributeurs GitHub </p> <p class='stat-figure'>172</p> <p class='stat-note'>En Juin 2019</p>
-  pTwitterFollowers: >
-    <p class='stat-label'>Followers sur Twitter </p> <p class='stat-figure'>17,588</p> <p class='stat-note'>En Juin 2019</p>
-  pSlackUsers: >
-    <p class='stat-label'> Utilisateurs Slack</p> <p class='stat-figure'>1,183</p> <p class='stat-note'>En Juin 2019</p>
+  pTotalTrades: Total des transactions
+  pTotalDate: Au cours du mois dernier
+  pDailyAverageTrade: Moyenne journalière des transactions
+  pSoftwareDownloads: Téléchargements du logiciel
+  pSoftwareDownloadsVersion: For v1.1.2 (approximation)
```

Still English. This looks like a decent translation from my time taking French.

```suggestion
  pSoftwareDownloadsVersion: Pour v1.1.2 (environ)
```

---

@m52go I think "approximátion" may be a better translation. @ArnaudFR86

---

Yes, I should have kept approximation. But in both cases, giving the exact version is not an approximation.
May be a better translation would be:  "environnement" which describe in computer field, hardware and software configuration specific to a computer.

---

@ArnaudFR86 The context here is that the figure (number of downloads) is an approximation, not a certain number.

---

OK, approximation is fine so :)